requirements.txt

```diff
@@ -10,13 +10,12 @@ colorama
 mlflow
 tensorflow
 python-multipart
+spacy
```

Comment:
Why adding those dependencies for your MVP ?

Reply:
You are not using spacy-contextualSpellCheck-pyspellchecker

Reply:
i think it was from lina's branch. it was mixed up a bit ...